about/introduction.rst

```diff
@@ -6,105 +6,98 @@ Introduction
 ::
 
     func _ready():
-        $Label.text = "Hello world!"
+        print("Hello world!")
```

Comment:
This is a tiny bit less Godot without showing off `$`, but also runnable in a new scene or project as is.
Also more traditional, I guess :) Not an important change.